website/versions.json

```diff
@@ -0,0 +1,3 @@
+[
+  "0.9.x"
```

Comment:
@darahayes FYI using 0.9.x here as we do not want to maintain patch releases docs.

Reply:
Good idea!